### ISSUE TITLE
Add aarch64 musllinux wheel for 0.8.0

### ISSUE THREAD

**Issue by rhuddleston** [openai/tiktoken#350]
0.7.0 had aarch64 musllinux wheel can the same be made for 0.8.0?

thanks!

**Comment by hauntsaninja** [comment]
Unfortunately, the QEMU emulation has become really flaky, it was painful even getting successful builds for the normal manylinux aarch64 wheels. If you can identify what's ailing the build, please open a PR and tag me, and I can release. Otherwise, I'm probably not going to do anything until Github has native aarch64 runners (I believe they said they'd make them available later this year)

**Comment by mwillfox** [comment]
For what it's worth, we also rely on aarch64 musllinux wheels for our Docker images. It looks like aarch64 runners are already available in [public preview](https://github.blog/news-insights/product-news/arm64-on-github-actions-powering-faster-more-efficient-build-systems/), and they should be taken out of preview something in Q1 2025. It might be worth a shot to try out the builds using them now?

**Comment by codefromthecrypt** [comment]
accidentally opened another issue [#386](https://github.com/openai/tiktoken/issues/386) not seeing this one. I closed in favor of focusing here. 🤞

**Comment by hauntsaninja** [comment]
Yeah, the emulation made life difficult, but Github now has arm runners, so I should be able to add a musl aarch wheel for next release

**Comment by codefromthecrypt** [comment]
@hauntsaninja gentle reminder that this would be awesome to have in. with grpcio 1.71.0 now having musl aarch64 wheels, having tiktoken could lead to fast and slim alpine build

**Comment by andreped** [comment]
> Unfortunately, the QEMU emulation has become really flaky, it was painful even getting successful builds for the normal manylinux aarch64 wheels. If you can identify what's ailing the build, please open a PR and tag me, and I can release. Otherwise, I'm probably not going to do anything until Github has native aarch64 runners (I believe they said they'd make them available later this year)

@hauntsaninja AFAIK, GitHub Actions now offers native ARM64 hosted runners, for macOS ([link](https://github.com/actions/runner-images?tab=readme-ov-file#available-images)), Linux ([link](https://github.blog/changelog/2025-01-16-linux-arm64-hosted-runners-now-available-for-free-in-public-repositories-public-preview/)), and Windows ([link](https://github.blog/changelog/2025-04-14-windows-arm64-hosted-runners-now-available-in-public-preview/)). So I would think creating the relevant wheels should be achievable. Thoughts?

**Comment by bdovaz** [comment]
> Yeah, the emulation made life difficult, but Github now has arm runners, so I should be able to add a musl aarch wheel for next release

Any news @hauntsaninja? Since then, there has been [another release](https://github.com/openai/tiktoken/releases/tag/0.11.0), but the arm wheel has still not been incorporated.

**Comment by hauntsaninja** [comment]
These wheels were built as part of 0.12.0
